backend/appwrite-functions/announcement/ancm-function/main.py

```diff
@@ -0,0 +1,209 @@
+import os
```

Comment:
Although what you implemented here looks good, I do not think we need this function at all, as discussed on Slack. All the functionalities that we require for announcements are already present in the frontend library. Create/Update/Delete can be done by admins only, as we define the collection in such a way that only admins have write access to it. Reading can be done by all users. Thus, this function only adds unnecessary complexity.

Reply:
In other words, as a frontend dev, we likely do not need this function, but it is a good starting point for future functions we might need to implement. Thus, I would likely add a comment in the file and/or Readme of the function indicating that the same thing can easier be achieved through the appwrite API itself, but the function could still be used and serves as a good example/starting point for future functions.